cocotb/binary.py

```diff
@@ -332,14 +352,10 @@ def get_buff(self):
                 buff = chr(val) + buff
         return buff
 
-    def get_hex_buff(self):
-        bstr = self.get_buff()
-        hstr = '%0*X' % ((len(bstr) + 3) // 4, int(bstr, 2))
-        return hstr
-
```

Comment:
Can you move this function back too?

Reply:
(the change from `self.get_buff()` to `self.buff` is still a good one)

Reply:
Oops. Missed that one!